common/scripted_triggers/wc_his_character_triggers.txt

```diff
@@ -129,12 +141,48 @@ is_kilrogg_trigger = {
 	exists = character:10005 # Kilrogg
 	this = character:10005 # Kilrogg
 }
+is_drekthar_trigger = {
+	exists = character:10100 # Drek'thar
+	this = character:10100 # Drek'thar
+}
+is_draka_trigger = {
+	exists = character:10020 # Draka
+	this = character:10020 # Draka
+}
+is_nazgrel_trigger = {
+	exists = character:10658 # Nazgrel
+	this = character:10658 # Nazgrel
+}
 
 # Dwarves
 is_magni_trigger = {
 	exists = character:17001 # Magni Bronzebeard
 	this = character:17001 # Magni Bronzebeard
 }
+is_muradin_trigger = {
+	exists = character:17002 # Muradin Bronzebeard
+	this = character:17002 # Muradin Bronzebeard
+}
+is_brann_trigger = {
+	exists = character:17028 # Brann Bronzebeard
+	this = character:17028 # Brann Bronzebeard
+}
+is_moira_trigger = {
+	exists = character:17029 # Moira Bronzebeard
+	this = character:17029 # Moira Bronzebeard
+}
+is_dagran_trigger = {
+	exists = character:12002 # Dagran Thaurissan
+	this = character:12002 # Dagran Thaurissan
+}
+is_kurdran_trigger = {
```

Comment:
What did you use Kurdran's for a reference? He's shown balding.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120846352-0cbc8080-c583-11eb-97ca-927a9eee8806.png)

</details>

Reply:
I wouldn't define Kurdran's hair as "balding" more that it's supposed to be a very long ponytail with everything else shaved intentionally. No such hairstyle exists, so I went for what I felt was the closest approximation.

Reply:
The hairstyle added, so it's resolved.